src/App.js

```diff
@@ -1,7 +1,7 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
 
-import { CharacterListView } from "./views";
-import "./styles/App.css";
+import { CharacterListView } from './containers';
```

Comment:
makes more sense to me as well -- I associate `Views` with mobile-app terminology personally, and it makes sense to me then in React Native but not here. this is just to expose students to multiple types of file structures/conventions throughout the unit so you don't get hung up on doing it "one way"